js/add_edit_buttons.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+    var settings = userProfile.addEditButtons;
+    setUserProfileButton();
+
+    function setUserProfileButton() {
+        var username = getUrlParam('username');
+        if (!username) {
+            return false;
+        }
+
+        if ($('#user-profile-btn').length !== 0) {
+            return false;
+        }
+
+        var $form = $('#edit_user_form');
+        if ($form.length === 0) {
+            return false;
+        }
+
+        if (typeof settings.existingProfiles[username] === 'undefined') {
+            var button = settings.addButton;
+            var url = settings.url + '&auto=1&user_profile_username=' + username + '&id=' + settings.nextProfileId;
```

Comment:
When _adding_ a new profile the username is not copied from the control center form to the new record in the user_profile redcap project.  I think the text 'user_profile_username' here needs to be replaced with the value of the config setting for user_profile_username.

Reply:
Now I'm forcing the module to be enabled in all projects. It solves the issue.